### DISCUSSION TITLE
Who wants to be hired? (Feb 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#33870] [Ideas]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

*If you're looking for last month's discussion, see #32938. We've had more [*success stories*](https://twitter.com/Nutlope/status/1475929984174895105) of developers being hired!*

If you're a company looking to post a role, use #33869.

**Comment by kellenwiltshire** [comment]
Hi Everyone,

My name is Kellen and I have been teaching myself web development since the end of 2020. I am eager to move forward with the next step of learning and hopefully get employment as a web developer full time!

Location: Ontario, Canada
Remote: Yes
Willing to relocate: No
Technologies: NextJS, React, TailwindCSS, Strapi, PostgresSQL, Emotion, Typescript, Squareup/Shopify
Résumé/CV: [Linkedin](https://www.linkedin.com/in/kellenwiltshire/ )
Website: [kellenwiltshire.com](https://kellenwiltshire.com)
Email: webdevelopment@kellenwiltshire.com

**Reply by TheThomasY** [reply]
Great portfolio! Very crisp design.

**Reply by kellenwiltshire** [reply]
@TheThomasY Thank you! I like yours as well!

**Reply by Mike-pw** [reply]
Noticed a typo "Optimization" , site looks nice!

**Reply by akhilkala** [reply]
Love the site. "Home" in the navbar is slightly misaligned.

**Reply by kellenwiltshire** [reply]
@MikeMelgren Nice catch! Thanks!

**Reply by kellenwiltshire** [reply]
@akhilkala That was a stylistic choice but maybe I need to rethink it! Thanks for the feedback!

**Comment by TheThomasY** [comment]
Hi I'm Tom,

I graduated with a Master's degree in Physics and since then have been learning Front End Development. Looking for an entry level role.

Location: London, UK
Remote: Yes
Willing to relocate: No
Technologies: Next.Js, React, TailwindCSS, Sass, Python
Resume/CV: [LinkedIn](https://www.linkedin.com/in/tomyoungdev/) 
Website: [Portfolio](https://tom-young-portfolio.vercel.app) 
Email: tomyoungdev@gmail.com

**Reply by Nutlope** [reply]
FYI @TheThomasY, the link to your website seems broken

**Reply by TheThomasY** [reply]
> FYI @TheThomasY, the link to your website seems broken

Thank you! Typo in the link but it's been fixed.

**Reply by Nutlope** [reply]
Awesome. Love the minimalistic design!

**Reply by TheThomasY** [reply]
Thank you!

**Reply by Mike-pw** [reply]
Also looking to make the jump from Physics -> Web Dev, just thought I'd say hello, and nice portfolio!

**Reply by Danm72** [reply]
Yo tom, I'm hiring for frontend devs at my digital agency in Dublin - you can apply using [this link ](https://apply.workable.com/mawla-team/j/A10FE4F0E3/)- ignore the senior/mid we've two spaces so open to junior

**Comment by thien-io** [comment]
Hello, I'm Thien. I've been studying modern web development and react since 2019. Imposter syndrome has always kept me from sharing my work as I was mostly following tutorials online. By practicing coding, and reading blog posts daily, I've gain confidence in my programming abilities, problem solving skills, ability to learn. I finally feel my skills are at a level where I can make meaningful contributes to a team. I dream to become a professional developer one day because I experienced how the web and friendly user interfaces can affect people's lives. The web is changing rapidly, and I want to be a part of making it better. 

Location: Connecticut, USA
Remote: Yes
Willing to relocate: Yes
Technologies: React, Javascript, Typescript, Nextjs, Postgres, MySql, Git, Node, Tailwindcss
Résumé/CV: https://thien.me
Website: https://thien.me
Email: thienjsx@gmail.com

Thank you for your interest.

**Comment by booleanrecep** [comment]
Hi, I'm Recep. Since 2019 my main focus is on front end development with React and its derivative tools. I am motivated to learn any other tech. Recently on the way learning Golang. I speak Turkish, English, German. I am looking for a full-time front end position and would also consider and feel motivated to do back end things.
Location: Istanbul /  Turkey 
Remote: Yes
Willing to relocate: Yes
Technologies: Vue, React, Next, Nuxt, TypeScript, any CSS framework, Node (willing to improve), , Golang(willing to improve), 
Resume/CV: [CV](https://docs.google.com/document/d/1bCS7AmJTvTgb57JCKVafu1zxq0wFMInNsKAZZpEb9LY/edit?usp=sharing)
Website: https://ahnung.vercel.app/
Email: recep.fed@gmail.com

**Comment by SSardorf** [comment]
Hey everyone! I am Simon - A Full-stack software developer and Data Scientist. As my master's degree in Data Science is coming to an end this summer, I am looking for exciting opportunities and thought I would start that journey here. I'll be looking for full-time/freelance/part-time positions.

Location: Copenhagen / Denmark
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next, Tailwind, Python, Django, FastAPI, PyTorch, Sklearn
Website: (Currently rebuilding - Feel free to ask for references of previous work)
Email: SSardorf@gmail.com

**Reply by Danm72** [reply]
Hey Simon, 

I'm hiring for frontend devs at my digital agency in Dublin - you can apply using [this link ](https://apply.workable.com/mawla-team/j/A10FE4F0E3/)- ignore the senior/mid we've two spaces so open to junior

**Comment by Dmytrocode** [comment]
I'm currently a Senior CS student in Canada. I'm looking to get an internship in any field. I am particularly interested in working with back-end technologies, but I have experience developing and deploying front-end applications with React and Python. I am willing to relocate anywhere. I am very motivated and I learn new things super fast! 

Dmytro.

Location: Saskatchewan, Canada
Remote: Yes
Willing to relocate: Yes
Technologies: .Net, Next, React, Python, Javascript,
Resume/CV:  https://www.linkedin.com/in/dmytro-kavetskyy-562198193/
Email: dmytrocontact@gmail.com

**Comment by developerisgood** [comment]
Hello, Everyone, This is Michael. Expert Web Front-End and Back-End Developer ready to take on your project.
You deserve a website that is built with security, responsiveness, functionality, and efficiency in mind. Over the last 10+ years, I have worked on all kinds of projects and I am able to build a web application from scratch very fast and with great efficiency or continue working on existing code with ease.
Location: Mexico
Remote: Yes
Willing to relocate: No
Technologies: React, Redux, Context-API, Ant Designs, Material UI,  Blueprint Js,  Typescript,  Styled Components, Storybook, NextJS, TypeScript
Resume/CV:  https://drive.google.com/file/d/1sdUb4xAeHCxnAzs4N0gLqqWA-InjoA7u/view?usp=sharing  
Website: https://michaeldev.herokuapp.com/
Email: micael.dev216@gmail.com

**Comment by talentjo** [comment]
Hello, This is Austin from USA. I am an experienced blockchain developer having vast knowledge of EVM (Ethereum Virtual Machine) and Solidity. I have worked on many complex block chain projects like Token development, Dapp (Decentralized Application) development.
Location: USA
Remote: Yes
Willing to relocate: No
Technologies: HTML5, SASS JavaScript SQL, MongoDB, ReactJS, Next, TypeScript, Vue2/Vue3, PHP, Laravel, Solidity.
Resume/CV: https://drive.google.com/file/d/1g6FKmjBLqccOk6710LNjDYEbVuC3d6gu/view?usp=sharing
Website: https://www.linkedin.com/in/codepro/
Email: farlight216@gmail.com

**Comment by HiImKevin** [comment]
Hi everyone,

I specialize in front-end development and web accessibility. I'm comfortable working as a sole developer, or in a smooth running team. Most recently I worked at Voya Financial helping build out their new design system and relaunch of the new voya.com. Since then I've moved on to branch out with my own dev agency.

Lately I've been working on some pretty cool headless CMS projects that lets content editors and marketers easily update their Next.js apps. I'd love to help out with your project!

Location: North America
Remote: Yes
Willing to relocate: No
Technologies: Next.js, React, Typescript, Tailwind, Headless CMS, AWS, Storybook, Web accessibility, A/B testing
Resume/CV: [LinkedIn](https://www.linkedin.com/in/kevin-wallace/)
Website: https://envizionlabs.com
Email: contact@envizionlabs.com

**Comment by ctnkaan** [comment]
Hey everyone.
I'm a community leader and a 3rd year Management Information Systems with a Software Engineering minor from Turkey. I'm looking for an internship in frontend web development, backend web development or DevOps. I'm open to work currently remotely and I can try to relocate at summer. 
Do check out my LinkedIn or Resume for more info!

Location: Turkey
Remote: Yes
Willing to relocate: For summer yes.
Technologies: HTML, CSS, JavaScript, TypeScript, React.js, Next, Node.js, Express, GraphQL, MongoDB, MySQL, PostgreSQL, Git. Docker, Azure
Website: https://www.linkedin.com/in/cetinkaantaskingenc/
Resume: https://drive.google.com/file/d/1nxCjnT4VHQ17Ua-pM6deCdtp6cQECYH6/view?usp=sharing
Email: [kaan.taskingenc@gmail.com](mailto:kaan.taskingenc@gmail.com)

**Comment by Rafi-99** [comment]
Hello everyone! 

My name is Rafi. I am an intrinsically motivated software engineer who loves building things for the web. I am looking to experience career growth in my next role and would love to put my Next.js skills to work!

Location: Minnesota, United States
Remote: Yes
Willing to relocate: Yes
Technologies: Java, JavaScript, React.js, Next.js, Node.js, Express.js, AWS, MongoDB
Resume/CV: [Md_Rafi_Résumé.pdf](https://github.com/vercel/next.js/files/7919866/Md_Rafi_Resume.pdf)
Website: [rafi-codes.dev](https://rafi-codes.dev)
Email: contact@rafi-codes.dev
LinkedIn: https://www.linkedin.com/in/rafi2018

Thank you for your time!

**Reply by anthonysidesap** [reply]
Whats ur pay rate

**Comment by abeciana1** [comment]
**Location:** currently based in Burlington, VT for now
**Remote:** Yes
**Willing to relocate:** Yes
Technologies: JavaScript, React, Gatsby, TypeScript, Next.js, Redux, GraphQL, Ruby, Ruby on Rails, PostgreSQL, HTML, CSS, Git, Netlify, Heroku, Vercel, and some Node.js/Express
**Resume/CV:** [Alex Beciana — Full Stack Software Engineer](https://drive.google.com/file/d/1fYN-OTgwVzbyUaraK6u_Xxp0q53OpvzC/view)
**Website:** https://alexbeciana.com
**Email:** alexander.beciana@gmail.com

**Comment by azizChebbi** [comment]
Location: Tunisia
Remote: Yes
Willing to relocate: Yes
Technologies: JavaScript, React, TypeScript, Next.js, Redux, HTML, CSS, Git, Node.js, Express, MongoDB
Resume/CV: https://www.linkedin.com/in/mohamed-aziz-chebbi-0387381b8/
Website: https://aziz-chebbi-54214.web.app/
Email: chebbim106@gmail.com

**Comment by Ben888GitHub** [comment]
Hi Everyone, I'm Benedict.

Creative Web Developer with experience working in front-end web and mobile applications development projects with Scrum/Agile methodology that empowers business growth.

Location: Singapore
Remote: Yes
Willing to relocate: No
Technologies: React, Firebase, Next.js, Redux, GraphQL, Bootstrap, Tailwind CSS, ChakraUI, Material UI, Web3.js, Ethers.js
Resume/CV: https://www.linkedin.com/in/benedict-ryan/
Website: https://benedict-ryan.vercel.app/
Email: [benedictryan80@gmail.com](mailto:benedictryan80@gmail.com)

**Comment by hey-code-everyday** [comment]
Hi, everybody.
This is James from Singapore.
I am a full-stack engineer with over 8 years of experience.
I have got hands-on experience in **front end development, backend development, blockchain development.**

Location: **Singapore**
Remote: **Yes**
Willing to relocate: **No**
Technologies: **TS/JS: React/Vue/Next/Nuxt/Nest/Node/Express/AWS/MUI/TailwindCSS/Web3/Solidity/Algorand**
Resume/CV: **https://www.linkedin.com/in/james-jin-a928281a2/**
Website: **https://www.linkedin.com/in/james-jin-a928281a2/**
Email: **jinzhengyou0903@gmail.com**

**Comment by G3root** [comment]
👋 Hey Everyone, I'm Nafees Nazik

- Location: Sri Lanka
- Remote: Yes
- Willing to relocate: Maybe
- Technologies: React, Next.js, React Native, GraphQL, Prisma, Tailwind CSS, CSS-in-JS, Trpc, RestApi, Typescript
- Resume/CV: https://drive.google.com/file/d/18Uh-cH9t8r0dUc_eok3vwpAQEtAyG4l4/view?usp=drivesdk
- Website: https://www.nafeez.xyz/
- Email: nafeesnazik21@gmail.com

**Comment by ibrahimyaacob92** [comment]
Hey Everyone, 
I'm Ibrahim Yaacob from Malaysia.
Full-stack engineer with focusing on Front-End, ReactJS/Next in particular. I'm here for the challenges to create a ground-breaking applications!

Location: **Malaysia**
Remote: **Yes**
Willing to relocate: **Depends**
Technologies: **TS/JS, Styled-Components, React/Next/Remix, Redux(RTK) Node/Express** 
Resume/CV: https://www.ibrahimyaacob.xyz
Website: https://www.ibrahimyaacob.xyz
Email: ibrahimyaacob92@yahoo.com

**Comment by sandrinjoy** [comment]
Hi I'm Sandrin Joy from India,

I graduated with a Bachelors degree in Computer Applications in 2021 and I am currently focusing on Front End Development espeically React and Tailwind

Location: India, IN
Remote: Yes
Willing to relocate: No
Technologies: Next.Js, React,Typescript, TailwindCSS, NodeJS, ExpressJS,Firebase
Resume/CV: [Google Docs](https://docs.google.com/document/d/1Ok9LwKjAHUw7CAFr0_QN35apdZnXi8OqRVI173lPyUA/edit?usp=sharing/)
Website: [sandrinjoy.com](https://sandrinjoy.com/)
Email: [sandrinejoy007@gmail.com](mailto:sandrinejoy007@gmail.com)

**Comment by Mike-pw** [comment]
Hello, I'm Mike from Canada,

I have a B.Sc. in Physics and have 10 years experience working in a remote team. I have been learning React/Next working on some freelance projects, and I am interested in furthering my practical knowledge of development. Looking for Part-Time / Casual work at the Entry/Junior level, preferably in a position where I am able to learn from people with more experience.

Location: Canada
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, GraphQL, TypeScript, Chakra UI
Resume/CV: https://techrez.io/resume/mike-melgren
Website: https://www.melgren.dev/
Email: mikemelgren@gmail.com

**Comment by Keshavdulal** [comment]
👋 Hi, I'm Keshav
- JavaScript Engineer (5.5Yrs)
- Experienced in building modern applications (PWAs & SPAs) on the web & mobile
- Proficient in rapid prototyping / MVP
- Enjoy working in teams

Location: Kathmandu, Nepal (GMT +5:45)
Remote: YES 🙌 
Willing to relocate: HELL YEAH
Technologies: React, Next.js, TailwindCSS, SASS, GraphQL, SQL, Mongo, Firebase, WordPress
Resume/CV: https://www.linkedin.com/in/keshavdulal
Website: https://keshavdulal.com || http://youraverageprogrammer.io
Email: keshav.dulal@gmail.com

**Comment by sup-Udh** [comment]
Hello Everyone! Hope y'all having a great day! 
’m a FullStack Developer with a passion for participate in every stage of digital product, from discovery to delivery. Helping to focus the business goals on users.


Location: Bangalore, India IST
Remote: Yes.
Willing to relocate: No

Resume/CV: https://udhaycodes.me

Technologies:
Typescript, JavaScript , next js, Tailwind css, Mongobd, SAAS, CSS , react js , node js , shell script ,postgressql
4 years Of Development! 

Website: https://udhaycodes.me  || https://www.polywork.com/udhay || https://www.linkedin.com/in/udhay-rajeev-42b193206/



Email:udhay1co.de@gmail.com 

Fun fact: Im 14..  :(

**Comment by amirardalan** [comment]
Hi, I'm Amir, a Front-end developer and UI designer based in Portland, Oregon. I have 10+ years experience building awesome web experiences for freelance clients, startups, agencies, and publicly traded corporations. I've worked within large code bases on teams utilizing Agile methodologies. I'm comfortable conducting code reviews, testing, and writing technical documentation. I'm proficient in communicating with team members at all levels of an org including non-technical stakeholders. I've spent the last several years working in Ecommerce using Salesforce Commerce Cloud – I'm currently seeking to pivot to a Next.js based role, preferably Mid/Senior Front-end. Thanks for your consideration.

Location: Portland, OR, USA
Remote: Yes
Willing to relocate: No
Technologies: Next.js, React, TypeScript, Emotion, Prisma, PostgreSQL, Three.js, Git
Resume/CV: https://amirardalan.com/amir-ardalan-resume.pdf
Website: https://amirardalan.com
Email: hi@amirardalan.com

**Reply by icld** [reply]
Hi Amir! Your site looks great.

**Comment by icld** [comment]


**Reply by amirardalan** [reply]
Great animations on your portfolio! Nice to see a fellow PDXer working with Next.js.

**Comment by JosefSaltz** [comment]
Hello,

My name is Josef Saltz, I have been running my own Web Development outfit at [thedotdevs.com](https://thedotdevs.com) where we build web applications for small businesses.

Location: Vancouver, WA USA
Remote: Yes
Willing to Relocate: No
Technologies: Next.js, React, Sass, Tailwind, Git, Express, Sequelize, Mongo, MySQL
Website: [thedotdevs.com](https://thedotdevs.com)
Resume: Available on website under my profile section.
email: [jps.webdev@gmail.com](maito:jps.webdev@gmail.com)

**Comment by CallmeHongmaybe** [comment]
Hello everyone, 
You can call me Dennis Kwan, and I've been focusing on my niche of making blogs from scratch where I either code it with Next.js or build it from the ground in Wordpress. 

Location: Ho Chi Minh City, Vietnam 
Remote: Yes
Willing to Relocate: Maybe 
Technologies: Next.js, React, Tailwind CSS, Material UI, Sanity CMS, Wordpress, and JAM stack. 
Website: [denniskwan.vercel.app](https://denniskwan.vercel.app/)
Resume on LinkedIn: https://www.linkedin.com/in/quan-hong/
email: quanvihong@gmail.com

**Comment by ariesclark** [comment]
I'm Aries, a 19 year old Canadian citizen with a burning passion for software development and improving the state of the web. I started software development when I was quite young — 13 years old in fact. In those 6 years I've done so many different things ranging from developing Minecraft mods in Java, writing quite a few horrible PHP websites, and all the way to where I am now. As of late, my focus has been on using modern languages like TypeScript to create web based applications and services with React and Next.js among other libraries.

I'm always open for an opportunity. Whether you have a question or just want to say hi, 
I'll try my best to get back to you! Take the chance and message me.

---

Location: Alberta, Canada
Remote: Yes
Willing to relocate: If given the right opportunity, yes.
Technologies: 
1. TypeScript / JavaScript (ES6+)
4. React and related libraries
5. Next.js (obviously ❤️)
6. Tailwind CSS 
7. and much much more.

Website: https://ariesclark.com/
Email: ``me@ariesclark.com``
